### DISCUSSION TITLE
Size of the IconButton, not the Icon

### DISCUSSION THREAD

**Discussion by ingdesarr1** [flet-dev/flet#4428] [Q&A] [answered]
Good morning. I want to know if it's possible add a option to change the size of the IconButton, not the size of the Icon. I observed that the icon_size option only changes the size of the icon, but the transparent area around the icon does not change, which generates unnecessary space.

Thanks

**Comment by ndonkoHenri** [comment, marked as answer]
[Uprgade](https://flet.dev/blog#how-to-upgrade) to the Flet v0.25.0 and use `IconButton.size_constraints`

Here's an example:
```python
import flet as ft


def main(page: ft.Page):

    page.add(
        ft.IconButton(
            ft.Icons.ADD,
            size_constraints=ft.BoxConstraints(min_width=200, min_height=200),
            icon_size=100,
        )
    )


ft.app(main)
``` 
![image](https://github.com/user-attachments/assets/8a50d7b1-6355-4682-9756-7e83867be48f)
